README.md

```diff
@@ -113,6 +113,16 @@ Perhaps Goby should be far easier for Rubyists to comprehend. You can use Ruby's
 - Flow control
     - `if`, `else`, `elsif`
     - `while`
+- Literals
+    - Numeric literal
+        - Integer: `100`, `-299`, `-0`
+        - Float: `3.14`, `-273.15`
+    - String literal: `"string"`, `'string'`, `:string`
+    - Symbol literal: `:email`, `{ tel: '9909-999-999' }
+        - Symbol literal is `String` class and just a convenient representation of string literals.
```

Comment:
I think saying `Symbol` is a convenient representation for `Rubyists` will be better, since not everyone who interested in Goby a Rubyist.